ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/editorhook/js/dorhook.js

```diff
@@ -38,6 +37,35 @@
             };
         };
 
+    var toolBar = document.getElementById("EditableToolbar");
+
+    if(Granite && Granite.isRDE){
+        toolBar.addEventListener('DOMNodeInserted',() => {
+            dorPreviewElement = document.querySelector(`[data-action="dorPreview"][icon="printPreview"]`);
+            
+            if (dorPreviewElement && !dorPreviewElement.disabled){
+                dorPreviewElement.disabled = true;
+
+                dorPreviewElement.removeAttribute("title");
+                var wrapper = document.createElement('span');
+                wrapper.setAttribute("class","dor-preview-button-wrapper");
+                dorPreviewElement.parentNode.insertBefore(wrapper, dorPreviewElement);
```

Comment:
can you pls share the screen shots also in this review?

Reply:
<img width="1728" alt="Screenshot 2023-03-28 at 11 43 46 AM" src="https://user-images.githubusercontent.com/56652547/228145079-4ab9d881-8c0e-4259-a9fa-8cfa3867a8ec.png">
<img width="1728" alt="Screenshot 2023-03-28 at 11 41 15 AM" src="https://user-images.githubusercontent.com/56652547/228145116-521364dd-4159-4944-8bce-acd725366be3.png">